ui/components/component-library/index.js

```diff
@@ -0,0 +1,25 @@
+export { AvatarAccount } from './avatar-account';
+export { AvatarFavicon } from './avatar-favicon';
+export { AvatarNetwork } from './avatar-network';
+export { AvatarToken } from './avatar-token';
+export { AvatarWithBadge } from './avatar-with-badge';
+export { BaseAvatar } from './base-avatar';
+export { Button } from './button';
+export { ButtonBase } from './button-base';
+export { ButtonIcon } from './button-icon';
+export { ButtonLink } from './button-link';
+export { ButtonPrimary } from './button-primary';
+export { ButtonSecondary } from './button-secondary';
```

Comment:
Yeah, could we get this in and add all the missing consts as apart of another housekeeping round? Otherwise this PR will just need to be constantly updated and waiting on others?